kubernetes/zenko/charts/zenko-queue/templates/configmap-config.yaml

```diff
@@ -21,7 +21,19 @@ data:
     done
     echo "Applying runtime configuration using {{ .Values.image }}:{{ .Values.imageTag }}"
     {{- range $n, $topic := .Values.topics }}
-    {{- if and $topic.partitions $topic.replicationFactor }}
+    {{- if and (and $topic.reassignPartitions $topic.replicationFactor) $topic.reassignPartitions }}
```

Comment:
is the extra `and` needed? As `$topic.reassignPartitions` is duplicated.

Reply:
oh, good catch. It's supposed to be $topic.partitions. That's why the extra `and`

Reply:
However, I just tested and `and` works with a multiple number of arguments.